packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -223,7 +223,9 @@ export interface TableOptions extends SchemaOptions {
    * This property can only be set if `encryption` is set to `TableEncryption.CUSTOMER_MANAGED`.
    *
    * @default - If `encryption` is set to `TableEncryption.CUSTOMER_MANAGED` and this
-   * property is undefined, a new KMS key will be created and associated with this table.
+   * property is undefined, a new encryption key will be created and associated with this table.
```

Comment:
```suggestion
   * property is undefined, a new KMS key will be created and associated with this table.
```
There is no reason to change this line. Saying that a KMS key will be created is the most accurate description of what CDK does for the user in this scenario. We need to inform the user that we will be creating resources in their account.